user/deployment/snaps.md

```diff
@@ -15,34 +15,51 @@ dist: xenial
 
 deploy:
   provider: snap
-  snap: my.snap
+  snap: my_*.snap
   channel: edge
   skip_cleanup: true
 ```
 {: data-file=".travis.yml"}
 
-To upload snaps from Travis CI, export a Snap Store login token. You can do this with the snapcraft command-line tool, once you have [enabled snap support](https://docs.snapcraft.io/core/install) on your system.
+The `snap` value should be a string that matches exactly one file when the deployment starts.
+If the name of the snap file is not known ahead of time, you can use a shell glob pattern, as shown
+in the example above.
+
+## Providing credentials to upload the snap
+To upload snaps from Travis CI, export a Snap Store login token, and provide it as an environment variable
+`$SNAP_TOKEN`.
+
+If you have not done so already, [enable snap support](https://docs.snapcraft.io/core/install) on your system.
 
 ```bash
 sudo snap install snapcraft --classic
 ```
 
-Login tokens can specify how, when, and where they can be used, thus minimising damage from compromise. For Travis CI, export a token that can only upload this snap to the channel you specified above (in this example, `edge`):
+Login tokens can specify how, when, and where they can be used, thus minimising damage from compromise.
+For Travis CI, export a token that can only upload this snap to the channel you specified above (in this example, `edge`):
 
 ```bash
 snapcraft export-login --snaps my-snap-name --channels edge -
 ```
 
 _Note: The final `-` requests the login be exported to stdout instead of a file. It is required._
 
-The token will be printed out. Copy and put it into the Travis CI environment variable `$SNAP_TOKEN`:
+The token will be printed out. 
+
+_Note: The `edge` channel is intended for the bleeding edge: your every commit to master will be built and uploaded._
+
+### Using the CLI cleint
+Using our [CLI client](https://github.com/travis-ci/travis.rb#readme), define `$SNAP_TOKEN`:
```

Comment:
```suggestion
Using our Travis CI, [CLI client](https://github.com/travis-ci/travis.rb#readme), define `$SNAP_TOKEN`:
```

Reply:
A good idea, but the comma is superfluous.